tests/std/tests/P1502R1_standard_library_header_units/env.lst

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+RUNALL_INCLUDE ..\..\..\universal_prefix.lst
+RUNALL_CROSSLIST
+PM_CL="/w14365 /D_ENFORCE_FACET_SPECIALIZATIONS=1"
+RUNALL_CROSSLIST
+PM_CL="/w14640 /Zc:threadSafeInit- /EHsc /std:c++latest"
+RUNALL_CROSSLIST
+PM_CL="/Zc:preprocessor /D_SILENCE_CXX17_STRSTREAM_DEPRECATION_WARNING"
+RUNALL_CROSSLIST
+PM_CL="/MD"
+PM_CL="/MDd"
+PM_CL="/MT"
+PM_CL="/MTd"
+# RUNALL_CROSSLIST
+# PM_CL=""
+# PM_CL="/analyze:only" # TRANSITION, works correctly but slowly

```

Comment:
is this transition a "let's remove this when analyze stops breaking header units quite so much"?

Reply:
No, it's the opposite - `/analyze` works equally well as plain C1XX compilation, but with significantly reduced throughput. Since it hasn't found any bugs, I've disabled it for now so we can iterate faster as we validate C1XX bugfixes. We can enable `/analyze` later to enhance the coverage.

Reply:
oh! my brain deleted the comment!